src/main/java/com/google/sps/servlets/TopChartsCluster.java

```diff
@@ -0,0 +1,66 @@
+
+package com.google.sps.servlets;
+import java.util.ArrayList;
+
+public class TopChartsCluster {
+    String title = "Top Charts";
+    String backgroundImage = "background-image.png";
+    ArrayList<Column> chart = new ArrayList<Column>();
+
+    
+
+    TopChartsCluster(){
+        App mockApp1 = new App(1, "mockApp1", "mockIcon1.png", "Fake Apps");
+        App mockApp2 = new App(2, "mockApp2", "mockIcon2.png", "Fake Apps");
+        App mockApp3 = new App(3, "mockApp3", "mockIcon3.png", "Fake Apps");
+        App mockApp4 = new App(4, "mockApp4", "mockIcon4.png", "Fake Apps");
+        App mockApp5 = new App(5, "mockApp5", "mockIcon5.png", "Fake Apps");
+        App mockApp6 = new App(6, "mockApp6", "mockIcon6.png", "Fake Apps");
+
+        ArrayList<App> topFree = new ArrayList<App>();
+        ArrayList<App> topGrossing = new ArrayList<App>();
+        ArrayList<App> topPaid = new ArrayList<App>();
+
+        Column topFreeColumn = new Column("Top Free", topFree);
+        Column topGrossingColumn = new Column("Top Grossing", topGrossing);
+        Column topPaidColumn = new Column("topPaid", topPaid);
+        topFree.add(mockApp1);
+        topFree.add(mockApp2);
+
+        topGrossing.add(mockApp3);
+        topGrossing.add(mockApp4);
+
+        topPaid.add(mockApp5);
+        topPaid.add(mockApp6);
+
+        chart.add(topFreeColumn);
+        chart.add(topGrossingColumn);
+        chart.add(topPaidColumn);
+    }
+   
+    
+}
+
+class Column{
+    String title;
+    ArrayList<App> apps;
+
+    Column(String title, ArrayList<App> apps){
+        this.title = title;
+        this.apps = apps;
+    }
+}
+
+class App{
```

Comment:
Move to `/models` package and make this a separate file, since it would be reused in other clusters as well.